### PR TITLE
[price-service] Add get_vaa_ccip endpoint

### PR DESCRIPTION
**Pull request by ali-bahjati**
This PR refactors `get_vaa` endpoint and adds another CCIP-compatible endpoint with the same functionality.

### CODE REVIEW

File: price-service/src/rest.ts [hunk 1]
```diff
@@ -54,6 +74,35 @@ export class RestAPI {
...
+            `${this.dbApiEndpoint}/vaa?id=${priceFeedWithoutLeading0x}&publishTime=${publishTime}&cluster=${this.dbApiCluster}`
+          ).then((res) => res.json()),
+        { retries: 3 }
+      )) as any[];
+
+      if (data.length > 0) {
+        vaa = {
+          vaa: data[0].vaa,
+          publishTime: Math.floor(
+            new Date(data[0].publishTime).getTime() / 1000
```
Comment (jayantk) [inline price-service/src/rest.ts:97]: what's this logic for?

Comment (jayantk) [reply price-service/src/rest.ts:97]: isn't this literally `data[0].publishTime * 1000` ?

**Review by jayantk — APPROVED** [review]
generally LGTM with a couple minor comments

File: price-service/src/rest.ts [hunk 2]
```diff
@@ -126,43 +175,102 @@ export class RestAPI {
...
+        if (
+          this.priceFeedVaaInfo.getLatestPriceInfo(priceFeedId) === undefined
+        ) {
+          throw RestException.PriceFeedIdNotFound([priceFeedId]);
+        }
+
+        const currentTime = Math.floor(new Date().getTime() / 1000);
+        if (publishTime > currentTime) {
+          throw RestException.PublishTimeInTheFuture();
+        }
```
[the remark this answers is not in the document]
Comment (ali-bahjati) [reply price-service/src/rest.ts:195]: I will remove it. :D

File: price-service/src/rest.ts (hunk 1 above)
Comment (ali-bahjati) [reply price-service/src/rest.ts:97]: Strangely it's not. The web-api returns the db column data which is a formatted string of time `2023-01-16T14:40:00.000Z` like this.

**Pull request merged**